com.bitwarden.desktop.yaml

```diff
@@ -5,10 +5,13 @@ sdk: org.freedesktop.Sdk
 base: org.electronjs.Electron2.BaseApp
 base-version: '22.08'
 command: bitwarden
+sdk-extensions:
+  # Node.js version: https://github.com/bitwarden/clients/blob/e2c25c6aad47d1e086ee8bb749e413cd93f89ca8/.nvmrc
+  - org.freedesktop.Sdk.Extension.node16
+  - org.freedesktop.Sdk.Extension.rust-stable
 finish-args:
   - --share=ipc
   - --share=network
-  - --socket=wayland
```

Comment:
I don't think removing wayland support is desirable, especially in this day and age, but maybe I'm misunderstanding what you're removing here?

Reply:
it has both x11 ansd wayland which is wrong. So something must be changed anyway.